tests/integration/linux_bridge_test.py

```diff
@@ -76,6 +77,14 @@ def bridge0_with_port0(port0_up):
         yield deepcopy(desired_state)
 
 
+@pytest.fixture
+def setup_vlan_iface(port0_up):
```

Comment:
The fixture represents vlan101 on port0, so it will be useful to name it as such.
`port0_vlan101`